### PR TITLE
Change network dashboards ownership to cabbage

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: scripts/sync-kube-mixin.sh
Comment (kopiczko) [inline scripts/sync-kube-mixin.sh:1]: Please note I changed this script and tested it by running it. But in the end I reverted the changes and updates owner tags manually because there were many other unrelated changes in the mixins. I think they should be merged in a separate PR.

**Review by Gacko — APPROVED** [review]
